envoy/upstream/upstream.h

```diff
@@ -149,7 +149,9 @@ class Host : virtual public HostDescription {
   /* is not meant to be routed to. */                                            \
   m(EXCLUDED_VIA_IMMEDIATE_HC_FAIL, 0x80)                                        \
   /* The host failed active HC due to timeout. */                                \
-  m(ACTIVE_HC_TIMEOUT, 0x100)
+  m(ACTIVE_HC_TIMEOUT, 0x100)                                                    \
+  /* The host is currently marked as draining by EDS */                          \
+  m(DRAINING_EDS_HEALTH, 0x200)
```

Comment:
Basically, it's OK for me to add more EDS statuses here. But we can give some better naming, like `EDS_STATUS_DRAINING`. Then in the future, we may add some new statuses like `EDS_STATUS_TIMEOUT`, etc.

Reply:
cc @htuch

Reply:
renamed to `EDS_STATUS_DRAINING`.